Packages/Newtonsoft.Json-for-Unity.Converters/UnityConverters/HideInJsonConverterSettings.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+
+namespace Newtonsoft.Json.UnityConverters
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class HideInJsonConverterSettings : Attribute { }
```

Comment:
Need a doc comment on this type to explain how it works and why you would use it